bin/cmd.js

```diff
@@ -35,7 +35,7 @@ const run = async (options) => {
   const deviceType = 'jetson-tx2'
   const stat = await statAsync(options.file)
 
-  if (!path.isAbsolute(options.output)) {
```

Comment:
I don't understand the commit log :)
What previous versions do you mean?

Reply:
@floion I can rewrite. It means:

```
!path.isAbsolute(options.output)
```
here `options.output` is `undefined`, and `path.isAbsolute()` cannot work with that.

What the patch does:

* does `options.output` have any value?
* if yes, check and adjust
* if not, nothing to check